src/System/Taffybar/Information/Brightness.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE OverloadedStrings #-}
+module System.Taffybar.Information.Brightness (readBrightnessValues) where
+
+import System.UDev.Context
+import System.UDev.Device
+import System.UDev.Monitor
+
```

Comment:
Can we get rid of these random spaces between imports. I don't really like grouping them in any way. They are always just sorted lexically.